docs/source/docs/level-up/intermediate_skills/09_data_exploration.rst

```diff
@@ -2,4 +2,56 @@
 Level 9: Dataset Explorartion from a Query Image/Text
 =====================================================
 
-TBD
+
+Datumaro support exploration feature to find out similar data for query among dataset. With query, the exploration result includes top-k similar data among dataset.
+Through this feature, you could figure out dataset property. You could check the visualization result of exploration using `Visualizer`.
+
+More detailed descriptions about explorer are given by :ref:`Explore`
+The Python example for the usage of explorer is described in `here <https://github.com/openvinotoolkit/datumaro/blob/develop/notebooks/07_data_explorer.ipynb>`_.
```

Comment:
Please check this comment

https://github.com/openvinotoolkit/datumaro/pull/960#discussion_r1172038374